docker/Dockerfile

```diff
@@ -112,12 +113,6 @@ RUN echo "Enable server extensions" \
   && jupyter serverextension enable --py --system jupyterlab_iframe \
   && echo done
 
-# Patch env when needed here
```

Comment:
This comment is referenced in README to guide future users/edits.

Reply:
Ah ok, I'll put it back in.